_partials/_livesync-console.md

```diff
@@ -0,0 +1,66 @@
+import PrereqCloud from "versionContent/_partials/_prereqs-cloud-only.mdx";
+import LivesyncLimitations from "versionContent/_partials/_livesync-limitations.mdx";
+import LivesyncConfigureSourceDatabase from "versionContent/_partials/_livesync-configure-source-database.mdx";
+import EarlyAccess from "versionContent/_partials/_early_access_2_18_0.mdx";
+
+## Prerequisites
+
+<PrereqCloud />
+
+- Install the [PostgreSQL client tools][install-psql] on your sync machine.
+
+  This includes `psql`, `pg_dump`, and `pg_dumpall`.
+
+<EarlyAccess />
+
+## Limitations
+
+<LivesyncLimitations />
+
+## Set your connection string
+
+This variable holds the connection information for the source database. In Terminal on your migration machine, 
+set the following:
+
+```bash
+export SOURCE="postgres://<user>:<password>@<source host>:<source port>/<db_name>"
```

Comment:
not required

Reply:
You need the $SOURCE param to run the commands in https://docs-dev.timescale.com/docs-3923-docs-rfc-add-console-live-sync-instructions/migrate/3923-docs-rfc-add-console-live-sync-instructions/livesync/#tune-your-source-database